mobile/src/main/java/org/horizontal/tella/mobile/media/MediaFileHandler.java

```diff
@@ -696,6 +696,9 @@ private static Bitmap flip(Bitmap bitmap, boolean horizontal, boolean vertical)
     }
 
     public static Observable<List<VaultFile>> getLastVaultFileFromDb() {
+        if (MyApplication.rxVault == null) {
+            return Observable.error(new NullPointerException("rxVault is null"));
```

Comment:
This was fixed using a reactive rx vault in the refactor it may cause some conflicts.